examples/third-party/react-datepicker/Datepicker.jsx

```diff
@@ -1,6 +1,5 @@
 import React from 'react'
 import { createField } from 'react-advanced-form'
-import moment from 'moment'
```

Comment:
Please, could you also remove `moment` from dev dependencies of react-advanced-form?
It's not used anywhere except of this example. Thanks :)